generators/action/index.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+var yeoman = require('yeoman-generator');
+var path = require('path');
+var ejs = require('ejs');
+var fs = require('fs');
+
+module.exports = yeoman.Base.extend({
+  constructor: function() {
+    yeoman.Base.apply(this, arguments);
+    this.argument('moduleName', {type: String, required: true});
+    this.argument('actionName', {type: String, required: true});
+  },
+
+  initializing: {
+    paths: function() {
+      fs.statSync(path.join(this.config.get('modulesPath'), this.moduleName), function(err) {
+        if (err) throw err;
+      })
+
```

Comment:
_To ensure that module is not automatically created if it does not exist_

Note to: @imranismail